src/filesys/file.c

```diff
@@ -17,16 +20,18 @@ struct file
 struct file *
 file_open (struct inode *inode) 
 {
-  struct file *file = calloc (1, sizeof *file);
+  struct file *file =  malloc(sizeof(file));   //calloc (1, sizeof *file);
```

Comment:
Calloc() and Malloc() are very similar. Except Calloc will initialize the allocated memory block (to 0 maybe) whereas malloc will NOT initialize the memory block